m2cgen/cli.py

```diff
@@ -0,0 +1,50 @@
+"""CLI for m2cgen.
+
+Example usage:
+    ./gen <path_to_file> --language java
+    gen --language java < <path_to_file>
+
+Model can also be piped:
+    cat <path_to_file> | gen --language java
+
+"""
+import pickle
+import argparse
+import sys
+
+import m2cgen
+
+
+LANGUAGE_TO_EXPORTER = {
+    "python": m2cgen.export_to_python,
+    "java": m2cgen.export_to_java,
+}
+
+
+parser = argparse.ArgumentParser(
+    description="Generate code in native language for provided model")
+parser.add_argument("infile", type=argparse.FileType("rb"), nargs="?",
+                    default=sys.stdin.buffer,
+                    help="File with pickle representation of the model")
+parser.add_argument("--language", dest="language", type=str,
```

Comment:
Looks great, but I'd say that the `package_name` is pretty important parameter in case of Java. Indentation can be added latter.